project.go

```diff
@@ -112,9 +131,42 @@ func (h *ProjectsHandler) List() ([]*Project, error) {
 		return nil, errors.New(response.Message)
 	}
 
+	for _, project := range response.Projects {
+		_, err = h.getProjectCA(project)
+		if err != nil {
+			return nil, err
+		}
+	}
+
 	return response.Projects, nil
 }
 
+func (h *ProjectsHandler) getProjectCA(project *Project) (*Project, error) {
```

Comment:
Seems weird to me that I can't just get the CACert by itself when using the Aiven client. I would implement the mixing of those information inside the terraform provider and keep the client as close as possible to the Aiven API.

Reply:
Good call @FranOis -- updated to reflect this suggestion